active-directory-new-domain/azuredeploy.json

```diff
@@ -253,10 +143,10 @@
             "name": "[variables('adRDPNAT')]",
             "properties": {
               "frontendIPConfiguration": {
-                "id": "[variables('adlbFEConfigID')]"
+                "id": "[concat(resourceId('Microsoft.Network/loadBalancers', variables('adLBName')), '/frontendIPConfigurations/', variables('adLBFE'))]"
```

Comment:
No, because /frontendIPConfigurations/adLBName needs to be appended onto the resourceId of the loadBalancer it belongs to